README.md

```diff
@@ -57,7 +57,9 @@ This chapter lists all available Metadata tags:
 - bundle-carefree-offline-SCU
 - bundle-carefree-online-SCU
 - custom-middleware-solution
-- portal
+- portal (for production and sandbox)
```

Comment:
I think we dont need the "portal (for production and sandbox)"

Therefore the text "concerns only sandbox)" and "(concerns only production)" can also be deleted.

Reply:
When I look on the whole picture I suggest to have this tags:
- portal
- sandbox
- production

(The "portal" is the old one.)
Why: The combination of other tags (eg -Middleware) with sandbox or production is also possible.